libs/packages/sam-formly/src/lib/formly/types/fileinfo.ts

```diff
@@ -4,51 +4,48 @@ import { FieldType } from '@ngx-formly/core';
 @Component({
   selector: 'sds-formly-field-file',
   template: `
-    <div class="grid-row grid-gap margin-top-1">
-      <div
-        *ngFor="
-          let option of (to.options | formlySelectOptions: field | async);
-          let i = index
-        "
-      >
-        <div
-          class="sds-card mobile-lg:grid-col"
-          [ngClass]="{ 'sds-card-selected': formControl.value == option.value }"
-        >
-          <input
-            type="radio"
-            [id]="id + '_' + i"
-            class="usa-sr-only usa-radio__input"
-            [name]="id"
-            [class.usa-input--error]="showError"
-            [attr.value]="option.value"
-            [value]="option.value"
-            [formControl]="formControl"
-            [formlyAttributes]="field"
-          />
-          <label [for]="id + '_' + i">
-            <div class="sds-card__header sds-card__header--center">
-              <h3 class="sds-card__title ">{{ option.label }}</h3>
-            </div>
-            <div class="sds-card__body sds-card__header--center ">
-              <span
-                class="bg-base-light padding-1 padding-left-2 padding-right-2"
-                >{{ option.value }}</span
-              >
-            </div>
-          </label>
+  <div class="grid-container width-tablet margin-top-1 margin-auto">
+  <div class="grid-row grid-gap">
+  <ng-container
+    *ngFor="
+      let option of (to.options | formlySelectOptions: field | async);
+      let i = index
+    "
+  > 
+  <div class="grid-col">
+  <label [for]="id + '_' + i">
+    <div
+      class="sds-button sds-button--icon-lg"
+      [ngClass]="{ 'sds-button--icon-lg--selected': formControl.value == option.value }"
+    >
+      <input
+        type="radio"
+        [id]="id + '_' + i"
+        class="usa-sr-only usa-radio__input"
+        [name]="id"
+        [class.usa-input--error]="showError"
+        [attr.value]="option.value"
+        [value]="option.value"
+        [formControl]="formControl"
+        [formlyAttributes]="field"
+      />
+  
+        <div class="sds-button__header">
+          <fa-icon [icon]="['sds',  to.options[i].icon]" class="text-base"  size="2x">
```

Comment:
minor suggestion here - option.icon should also work rather than to.options[i].icon